frb/builds/build_hosts.py

```diff
@@ -182,11 +198,12 @@ def build_host_121102(build_photom=False, build_cigale=False, use_orig=False):
     if use_orig:
         host121102.derived['M_r'] = -17.0  # AB; Tendulkar+17
         host121102.derived['M_r_err'] = 0.2  # Estimated by JXP
-        host121102.derived['SFR_nebular'] = 0.23  # MSun/yr; Tendulkar+17
+        host121102.derived['SFR_nebular'] = 0.15  # MSun/yr from new dust-corrected H-alpha; (0.23 from Tendulkar+17)
```

Comment:
`use_orig=False`, i.e. this is not called.
Is that what you expected?

Reply:
Yes, have included the SFR measurements from the H-alpha line flux further down directly so this should not be called. Have just added the original value now.